backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Controllers/UserController.cs

```diff
@@ -57,5 +59,15 @@ public async Task<ActionResult<List<UserShortInfoDto>>> GetSuitableUsers([FromBo
             return await _userService.GetFilteredUsers(userFilter);
         }
 
+        [HttpPut("makeAdmin/{userId}")]
+        public async Task<ActionResult<UserDto>> MakeAdminAsync(int userId, string secret)
```

Comment:
Can you add secret to query?

Reply:
Can you explain in more detail what should I change